src/lib/table/table-foundation.ts

```diff
@@ -979,6 +986,16 @@ export class TableFoundation implements ITableFoundation {
     this._headCellMouseDownIndex = undefined;
   }
 
+  private _onSortableHeadCellKeydown(evt: KeyboardEvent): void {
```

Comment:
Should this use click events instead of keydown and mousedown now that it's a button?